docs/user/access-control.md

```diff
@@ -40,39 +40,32 @@ Kubeapps.
 
 #### Read access to Applications within a namespace
 
-In order to list and view Applications in a namespace, apply the
-`kubeapps-applications-read` ClusterRole in the desired namespace and the
-`kubeapps-tiller-state-read`:
+In order to list and view Applications in a namespace, apply the `view` ClusterRole
+in the desired namespace. The `view` ClusterRole should be available in most
+Kubernetes distribution, you can find more information about that role [here](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles).
 
 ```
-kubectl create -n default rolebinding example-kubeapps-applications-read \
-  --clusterrole=kubeapps-applications-read \
+kubectl create -n default rolebinding example-view \
+  --clusterrole=view \
   --serviceaccount default:example
 ```
 
-Note: This will give read access to all the resources in the default namespace.
```

Comment:
isn't this still true?

Reply:
It doesn't give read access to "all", just a big set. Anyway I removed the `Note` because the role it's explained before with the link to the kubernetes documentation.

Reply:
Okay, that sounds fine, thanks!